packages/validator/src/validator.ts

```diff
@@ -85,6 +86,10 @@ export class Validator {
     } = opts;
     const config = createIBeaconConfig(dbOps.config, genesis.genesisValidatorsRoot);
     this.controller = new AbortController();
+    // We set infinity to prevent MaxListenersExceededWarning which get logged when listeners > 10
+    // Since it is perfectly fine to have listeners > 10
+    setMaxListeners(Infinity, this.controller.signal);
```

Comment:
Can you commit as comment the stack trace proving that this very specific event emitter gets more that 10 listeners? Same for below

Reply:
I removed the changes for the abort controller, as I could find a situation where it triggered the warning. I left the one for ValidatorEventEmitter, even though I could not recreate the situation that triggered it, I did observe by chance back in January. See  #3569. The log reproduced here

```
(node:15159) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 chainHead listeners added to [ValidatorEventEmitter]. Use emitter.setMaxListeners() to increase limit
 ```